src/components/tailwind/SitewideHeader/index.js

```diff
@@ -0,0 +1,109 @@
+import React, { Fragment } from 'react'
```

Comment:
More work to do on this component in https://app.shortcut.com/larder/story/4282/migrate-the-sitewide-header